src/source.ts

```diff
@@ -251,7 +287,7 @@ abstract class SourceContractItem implements Linkable {
   @memoize
   get args(): SourceTypedVariable[] {
     return SourceTypedVariableArray.fromParameterList(
-      this.astNode.parameters,
+      ('parameters' in this.astNode)?this.astNode.parameters:{parameters: []},
```

Comment:
I'd prefer to rollback this line, and instead change the type of `astNode` to be `FunctionDefinition | EventDefinition | ModifierDefinition`.